src/main/features/first-contact/views/start.njk

```diff
@@ -22,7 +22,7 @@
 
       <p>{{ t('They can use a CCJ to get the money from you, for example by applying to the court to send bailiffs to your home or business.') }}</p>
 
-      <p>{{ t('A CCJ may make it harder to get credit, such as a mobile phone contract, credit card or mortgage.') }}</p>
+      <p>{{ t('A CCJ may make it harder to get credit, such as a phone contract, credit card or mortgage.') }}</p>
```

Comment:
this is content and shouldn't be changed. should only be changing variable, paths and file names